framework/source/class/qx/test/Bootstrap.js

```diff
@@ -292,7 +292,7 @@ qx.Class.define("qx.test.Bootstrap",
 
           stopEngine : function()
           {
-            var ret = arguments.callee.base.call();
+            var ret = qx.test.Bmw.prototype.stopEngine.base.call();
```

Comment:
Is this the same as the following?
```
var ret = this.base(arguments);
```

.call() with no first argument means that the context is explicitly excluded. That sounds wrong to start with, unless there is no reference to `this` in the `...stopEngine.base` method...